### PR TITLE
Added contact me form to my homepage

### PR DESCRIPTION
**Pull request by mershark**
In this task, Together with my partner @Young-Gaiu, we added a new section to my portfolio homepage which is the "contact me form".
The form is able to receive and record input into my formspree account.

The only files/directories updated in this task are:
- The images directory
- Style.css file
- Index.html file
- README.md file

### CODE REVIEW

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

👋 Hello! Thank you for your submission. I noticed that you have he requirements for the project, which is great. However, I would like to remind you that this milestone is a pair-programming activity. As stated in the guidelines, it is important to collaborate with your coding partner and ensure that both of you have actively contributed to each project.

Please make sure to coordinate with your coding partner and ensure that both of you have actively participated in the project and have left your comments and contributions accordingly.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -149,5 +149,30 @@ <h4 class="skills-h4">Skills</h4><hr class="skills-underline">
...
+      <ul class="form-ul">
+        <li>
+          <input type="text" name="full name" id="form-name" placeholder="First and last name" required maxlength="30">
+        </li>
+        <li>
+          <input type="email" name="Email" id="form-email" placeholder="Email (email@gmail.com)" required="">        
+        </li>
+        <li>
+          <input type="message" name="message" id="form-message" placeholder="Write your message here" required maxlength="500">        
+        </li>
```
Comment (usorfaitheloho) [inline index.html]: Kindly use a textarea attribute for the message field as stated in your form. The `textarea` attribute is used to define multi-line text input control and is often used in a form, to collect user inputs like comments or reviews 😄
You can check [this article](https://www.w3schools.com/tags/tag_textarea.asp) for more details

File: style.css [hunk 2]
```diff
@@ -945,3 +945,157 @@
...
+  padding-right: 24px;
+}
+
+.form-ul {
+  list-style-type: none;
+}
+
+.contact-me-form {
+  padding-top: 52px;
+}
```
Comment (usorfaitheloho) [inline style.css]: Please make the following design changes to improve your projects UI and to have a better design which looks similar to the Figma template given 😄 :

- Kindly add the background images present in the design given to improve your form UI
- Kindly adjust the alignment of the text `If you have an application you are interested in developing ...`
- please use the right font-color property for the placeholder text in your form which is `#172B4D` and also reduce the padding in the form to look close to what is in the design
- your `submit button` looks really different from the design you were given, it has border-color and doesn't use the right font-color and size as shown in the design. Please also remember that your button should also use the right button interaction given for your design 😄 

Figma template :

image

Your design :
image

 Button interaction :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @mershark ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Used Formspree correctly ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by usorfaitheloho — APPROVED** [review]
Hi @mershark  , 
# STATUS: APPROVED! 🥇 💯 
- [x] Very Nice Design, Keep it up!!
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**